tests/main/snap-userd-desktop-app-autostart/task.yaml

```diff
@@ -33,4 +31,4 @@ execute: |
     done
     test -e ~/snap/test-snapd-xdg-autostart/current/foo-autostarted
 
-    test "$(get_journalctl_log_from_cursor "$cursor" | grep -c foo.desktop)" -gt 0
+    test "$("$TESTSTOOLS"/journal-state get-log_from_cursor "$cursor" | grep -c foo.desktop)" -gt 0
```

Comment:
shouldn't this be 

```suggestion
    test "$("$TESTSTOOLS"/journal-state get-log-from-cursor "$cursor" | grep -c foo.desktop)" -gt 0
```
for consistency's sake ?

Reply:
Nice catch, thans for the review!!